packages/esm-patient-labs-app/src/lab-orders/add-lab-order/add-lab-order.scss

```diff
@@ -34,6 +34,7 @@
       order: 1;
       margin-right: layout.$spacing-03;
       margin-left: 0 !important;
+      fill: currentColor !important;
```

Comment:
Yeah, I've got to sort this out. The broad issue is that we're using CSS to try to propagate the color, but Carbon tends to use more "specific" selectors to color things. And while it would be possible to use `!important` in the styleguide itself, it would make it impossible to override in parent CSS when appropriate...